src/test/ui/recursion_limit/zero.stderr

```diff
@@ -0,0 +1,10 @@
+error: recursion limit reached while expanding the macro `test`
```

Comment:
The PR test pipeline fails on this test. On my local machine the test passes, on the Azure machine, the error message is 
```
error: recursion limit reached while expanding `test!`
```
Missing "the macro" and inserting a "!". Any suggestions on what's different?

Reply:
It's likely that the error message changed in the latest version of rust. You can either rebase, rebuild and rebless, or you could just copy the CI output into your branch.